src/config.py

```diff
@@ -5,14 +5,21 @@
 
 @section("redis")
 class RedisConfig(Config):
+    # Hostname of a configured redis instance.
     host = key(cast=str, required=False, default="localhost")
+    # Port of a configured redis instance.
     port = key(cast=int, required=False, default=6379)
 
 
 @section("mquery")
 class MqueryConfig(Config):
+    # URL to a UrsaDB instance.
     backend = key(cast=str, required=False, default="tcp://127.0.0.1:9281")
+    # List of plugin specifications separated by comma, for example
+    # "plugins.archive:GzipPlugin, plugins.custom:CustomPlugin"
```

Comment:
```suggestion
    # "plugins.archive:GzipPlugin,plugins.custom:CustomPlugin"
```
I guess?

Reply:
Both will work (spaces are stripped), I like it with space a bit more (looks prettier)